.gitlab/kernel_matrix_testing/common.yml

```diff
@@ -101,6 +101,7 @@
     - !reference [.write_ssh_key_file]
   tags: ["arch:amd64"]
   script:
+    - source /root/.bashrc
```

Comment:
You might want to move this at the beginning of the `before_script`, it's not rare that we rely on `aws` in some setup jobs, but this isn't blocking at all

Same remark for other `source`s in this PR

Reply:
Thanks for making the change :)